src/main/java/com/icthh/xm/ms/entity/service/LinkService.java

```diff
@@ -120,6 +122,7 @@ public Link findOne(Long id) {
 
     @FindWithPermission("LINK.SOURCE.GET_LIST")
     @Transactional(readOnly = true)
+    @PrivilegeDescription("Privilege to get all the sources")
```

Comment:
suggest descriptions:
`Privilege to get all the source links by target entity id and typeKeys`